docker-compose.yml

```diff
@@ -111,6 +111,41 @@ services:
       # <host_machine_directory>:<container_directory> – mirror bidirectional path in local context with path in Docker container
       - ./logs/backend:/logs/backend
 
+  ########################################################
+  # DHT-NODE #############################################
+  ########################################################
+  dht-node:
+    # name the image so that it cannot be found in a DockerHub repository, otherwise it will not be built locally from the 'dockerfile' but pulled from there
+    image: gradido/dht-node:local-production
+    build:
+      # since we have to include the entities from ./database we cannot define the context as ./backend
+      # this might blow build image size to the moon ?!
+      context: ./
+      dockerfile: ./dht-node/Dockerfile
+      target: production
+    networks:
+      - internal-net
+    ports:
+      - 4000:4000
```

Comment:
thd dht modul should use 5000, because backend uses the port 4000

Reply:
```suggestion
      - 5000:5000
```